scripts/monorepo/index.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   findRepoDeps: require('./findRepoDeps'),
   getAllPackageInfo: require('./getAllPackageInfo'),
   isConvergedPackage: require('./isConvergedPackage'),
+  isCompatibilityPackage: require('./isCompatibilityPackage'),
```

Comment:
you should update the types file in the `monorepo` directory

Reply:
Thanks - somehow missed it working in JS